extensions/binding/org.eclipse.smarthome.binding.dmx/build.properties

```diff
@@ -0,0 +1,11 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/,\
+               lib/
+               lib/ola-java-client-0.0.3.jar,\
```

Comment:
Since you have the `lib` directory in the `bin.includes` you do not have to specify the jar files here, so please remove them and keep the `lib/` entry.